sdk/helper/automatedrotationutil/fields.go

```diff
@@ -102,23 +102,41 @@ func (p *AutomatedRotationParams) HasNonzeroRotationValues() bool {
 
 // AddAutomatedRotationFields adds plugin identity token fields to the given
 // field schema map.
-func AddAutomatedRotationFields(m map[string]*framework.FieldSchema) {
+func AddAutomatedRotationFields(m map[string]*framework.FieldSchema, group ...string) {
```

Comment:
I understand why we're using the variadic operator here, but it doesn't feel great if there's only ever going to be one value supplied.

Reply:
Ahh I see and then the `NewFunc` could be called directly when needing to use something other than the default group. I haven't worked with Go enough to have an opinion on whether or not that would be a better experience. What do you think? I'm happy to update if so.

Reply:
I think if it were totally up to me, I would add the new function, note on the old one in comments that it's "deprecated", and some kind of note that if someone needed to write this kind of function for another "util" package, they should use the "group-included" pattern.

Might not be a bad idea to add a `const GroupDefault = "default"` somewhere if we can figure out a reasonable place for it.

Reply:
That all sounds good to me thanks! I pushed a commit with the changes and added some test coverage. Let me know what you think.